magma/tuple.py

```diff
@@ -430,6 +430,19 @@ def value(self):
         return namedtuple(**dict(zip(self.keys(),ts)))
 
 
+class AnonProductMeta(ProductMeta):
```

Comment:
This type relationship is backward.

Reply:
@splhack can you update this to match the new `hwtypes` hierarchy? See https://github.com/leonardt/hwtypes/pull/123 (basically, anon product is a parent of product)